doc/cookbook/namedRoutes/NamedRoutes.lhs

```diff
@@ -0,0 +1,383 @@
+# Record-based APIs: the nested records case
```

Comment:
nested API*

It does indeed result in "nested records", but this merely mirrors the structure of the API, which is the central element that should be in the title IMO